server/src/main/java/io/deephaven/server/table/ops/UpdateByGrpcImpl.java

```diff
@@ -0,0 +1,231 @@
+package io.deephaven.server.table.ops;
+
+import io.deephaven.api.updateby.BadDataBehavior;
+import io.deephaven.api.updateby.ColumnUpdateClause;
+import io.deephaven.api.updateby.EmaControl;
+import io.deephaven.api.updateby.UpdateByClause;
+import io.deephaven.api.updateby.UpdateByControl;
+import io.deephaven.api.updateby.spec.CumMinMaxSpec;
+import io.deephaven.api.updateby.spec.CumProdSpec;
+import io.deephaven.api.updateby.spec.CumSumSpec;
+import io.deephaven.api.updateby.spec.EmaSpec;
+import io.deephaven.api.updateby.spec.FillBySpec;
+import io.deephaven.api.updateby.spec.TimeScale;
+import io.deephaven.api.updateby.spec.UpdateBySpec;
+import io.deephaven.proto.backplane.grpc.BatchTableRequest;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest.Operation;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest.Operation.Column;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest.Operation.Column.Spec;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest.Operation.Column.Spec.CumulativeMax;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest.Operation.Column.Spec.CumulativeMin;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest.Operation.Column.Spec.CumulativeProduct;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest.Operation.Column.Spec.CumulativeSum;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest.Operation.Column.Spec.Ema;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest.Operation.Column.Spec.Fill;
+import io.deephaven.proto.backplane.grpc.UpdateByRequest.Options;
+import io.deephaven.qst.table.TableSpec;
+import io.deephaven.qst.table.UpdateByTable;
+import io.deephaven.qst.table.UpdateByTable.Builder;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.math.MathContext;
+import java.math.RoundingMode;
+
+@Singleton
+public final class UpdateByGrpcImpl extends GrpcQstTableOperation<UpdateByRequest> {
+
+    @Inject
+    public UpdateByGrpcImpl() {
+        super(BatchTableRequest.Operation::getUpdateBy, UpdateByRequest::getResultId, UpdateByRequest::getSourceId);
+    }
+
+    @Override
+    TableSpec createTableSpec(UpdateByRequest request) {
+        return adapt(request);
+    }
+
+    private static UpdateByTable adapt(UpdateByRequest request) {
```

Comment:
Requesting a method name other than "adapt" for the the next 100+ lines of this class